src/components/Chips/PostDirectionChip/PostDirectionChip.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { PostDirectionChip } from './PostDirectionChip';
+
+describe('PostDirectionChip', () => {
+  it('should render the chip with correct class', () => {
+    const { getByTestId } = render(<PostDirectionChip />);
+    expect(getByTestId('chip')).toHaveClass(
+      'PostDirectionChip-directionChip-1',
+    );
+  });
+});
+
+describe('events', () => {
+  it('should delete the chip', () => {
+    const handleDeleteMock = jest.fn();
+    const { getByTestId, unmount, container } = render(
+      <PostDirectionChip handleDelete={handleDeleteMock} />,
+    );
+    const deleteIcon = container.querySelector('.MuiChip-deleteIcon');
+    const chip = getByTestId('chip');
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    fireEvent.click(deleteIcon);
+    expect(handleDeleteMock).toHaveBeenCalledTimes(1);
+    unmount();
```

Comment:
What's the point of lines 26-27? If you unmount it, of course it will not be in the document

Reply:
Fixed with commit 3f6b211